src/emit/soroban/target.rs

```diff
@@ -51,7 +72,28 @@ impl<'a> TargetRuntime<'a> for SorobanTarget {
         function: FunctionValue<'a>,
         ns: &ast::Namespace,
     ) {
-        unimplemented!()
+        emit_context!(binary);
```

Comment:
Why is `emit_context` necessary here?

Reply:
It's for `i64_const!(2).into(),` in this [line](https://github.com/salaheldinsoliman/solang/blob/84d5f7db6b5d3010b062a075b4c50ea43036f610/src/emit/soroban/target.rs#L88). 
I can use:
`binary.context.i64_type().const_int(2, false).into()` though